src/autora/experimentalist/model_disagreement/__init__.py

```diff
@@ -9,6 +9,133 @@
 from autora.utils.deprecation import deprecated_alias
 
 
+def score_sample_cd(
```

Comment:
This is minor but I'm wondering whether the function name is could be more self-explanatory? E.g., score_sample_custom_distance?

Reply:
renamed it to ..._custom_distance